openapi.yaml

```diff
@@ -1,279 +1,914 @@
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+  /services:
+    get:
+      tags:
+        - services
+      summary: Get services
+      description: Gets all `Service` objects. Can be filtered by parameters.
+      operationId: getServices
+      parameters:
+        - in: query
+          name: filter[search]
+          description: Query to search through buildings. Looks through the building name, abbreviation.
+          required: false
           schema:
-            $ref: '#/definitions/PetResult'
+            type: string
+            example: dixon
+        - $ref: '#/components/parameters/isOpen'
+        - $ref: '#/components/parameters/pageNumber'
+        - $ref: '#/components/parameters/pageSize'
+      responses:
+        '200':
+          description: Successful response
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ServiceSetResult'
         '400':
-          $ref: '#/responses/400'
-        '409':
-          description: Conflict
-          schema:
-            $ref: '#/definitions/ErrorResult'
+          $ref: '#/components/responses/400'
         '500':
-          $ref: '#/responses/500'
-  /pets/{id}:
+          $ref: '#/components/responses/500'
+  /services/{serviceId}:
     get:
-      summary: Info for a specific pet
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+        - services
+      description: Finds service by ID
+      operationId: getServiceById
       parameters:
-        - name: id
+        - name: serviceId
           in: path
-          description: Unique Pet ID, UUID format
+          description: ID of service
           required: true
-          type: string
+          schema:
+            type: string
       responses:
         '200':
           description: Successful response
-          schema:
-            $ref: '#/definitions/PetResult'
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ServiceResult'
         '404':
-          description: PetResource not found
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+  /services/{serviceId}/locations:
+    get:
+      tags:
+        - services
+        - locations
+      description: Finds locations related to a service
+      operationId: getServiceLocationsById
+      parameters:
+        - name: serviceId
+          in: path
+          description: ID of service
+          required: true
           schema:
-            $ref: '#/definitions/ErrorResult'
+            type: string
+      responses:
+        '200':
+          description: Successful response
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ServiceLocationSetResult'
```

Comment:
open to suggestions on this, but I thought it made sense for this endpoint to return a response without included or meta. 
* Leaving out included because we're only focused on the relationship between this service and its locations, not others services/locations
* Leaving out meta because this endpoint doesn't use pagination

I made the same change in the locations/:id/services endpoint. Open to discussion on this but it makes sense to me.

Reply:
Sounds good to me. @Ruefa @kalcicd since you also review this, any thoughts?